gcsfs/tests/test_core.py

```diff
@@ -471,6 +471,23 @@ def test_get_put(consistency):
             assert gcs.cat(TEST_BUCKET + "/temp") == data
 
 
+@pytest.mark.skipif(ON_VCR, reason="async fail")
```

Comment:
Does this, in fact, fail with VCR?

Reply:
I was not able to test this - I think it runs through a nearly identical code path to tets_get_put which had this mark though which is why i included it